### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by cwaku**
- I built the project schema.
  - Created and run the necessary migration files.
  - Table and column names match the ERD diagram.
    - note: photo for users table is a link to an image
  - Foreign keys included.
  - All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20220401131950_create_posts.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commets_counter, default: 0
+      t.integer :likes_counter, default: 0
```
Comment (SajjadAhmad14) [inline db/migrate/20220401131950_create_posts.rb:9]: Please add migration for `author_id` as foreign key in `posts` table as shown in [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ricky, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/schema.rb [hunk 2]
```diff
@@ -10,10 +10,29 @@
...
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_likes_on_author_id"
+    t.index ["post_id"], name: "index_likes_on_post_id"
+  end
+
   create_table "posts", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
```
Comment (julie-ify) [inline db/schema.rb:38]: - [ ] **[OPTIONAL]** Kindly run `rails db:migrate` to update your schema with all columns created in all your migrations.

**Review by julie-ify — APPROVED** [review]
Hi @cwaku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 2 above)
Comment (cwaku) [reply db/schema.rb:38]: Thanks, I'll do that

**Pull request merged**